selfdrive/ui/qt/maps/map.cc

```diff
@@ -24,16 +24,8 @@ const float MAP_SCALE = 2;
 
 const QString ICON_SUFFIX = ".png";
 
-MapWindow::MapWindow(const QMapboxGLSettings &settings) :
-  m_settings(settings), velocity_filter(0, 10, 0.05) {
-  sm = new SubMaster({"liveLocationKalman", "navInstruction", "navRoute"});
-
-  // Connect now, so any navRoutes sent while the map is initializing are not dropped
-  sm->update(0);
```

Comment:
make sure the UIState SubMaster effectively does this too

Reply:
It should always, we go onroad from UIState's SubMaster data, and we only called this sm.update() here when we went onroad.